src/main/java/core/basesyntax/strategy/operations/BalanceOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy.operations;
+
+import core.basesyntax.dao.FruitsDao;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.impl.FruitServiceImpl;
+
+public class BalanceOperation implements OperationHandler {
+
+    @Override
+    public void runOperation(FruitsDao fruitsDao, String fruitName, int quantity) {

```

Comment:
we should not pass FruitsDao as method incoming parameter, lets create class level field for FruitsDao in BalanceOperation class and initialize it with constructor
(same for other operation handlers)